edgecast/waf/custom_rule.go

```diff
@@ -247,3 +256,24 @@ func (svc *WAFService) GetAllCustomRuleSets(accountNumber string) ([]CustomRuleS
 
 	return *customRuleSets, nil
 }
+
+// Deletes a custom rule.
+func (svc *WAFService) DeleteCustomRuleSet(accountNumber string, customRuleId string) (*DeleteCustomRuleResponse, error) {
```

Comment:
We can discuss about naming convention but golang recommend to use customRuleID instead of customRuleId.

Reply:
agreed. updated to golang conventions.